lib/crowbar/client/command/proposal/create.rb

```diff
@@ -104,6 +104,11 @@ def from_file
               err "Failed to process file"
             end
 
+            if json["id"] != args.proposal
+              args.proposal = json["id"]
+              say "Given proposal name and file proposal name do not match, using id(#{json["id"]})"
```

Comment:
maybe simplify to

```ruby

   say "Using #{json["id"]} as proposal name"
```

since there no way to not do that anymore? what about options[:merge] ? should that be disabled?